packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1389,6 +1389,7 @@ base class PipelineOwner with DiagnosticableTreeMixin {
         );
         _semanticsOwner = SemanticsOwner(onSemanticsUpdate: onSemanticsUpdate!);
         onSemanticsOwnerCreated?.call();
+        flushSemantics();
```

Comment:
all embedding including our desktop has a trigger in the accessibility bridge to enable semantics when os query the a11y tree. For platform that has merged thread, that trigger will call this method synchronously.

This method creates semantics tree and sends update to embedding synchronously.